### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by slaffcheff** [slaffcheff/cliarmy#5]
Logo + eventually cover picture for the header. Will discuss. Mandatory task!

**Comment by teddy931130** [comment]
. --- .
		   /        \
		  |  O  _  O |
		  |  ./   \. |
		  /  `-._.-'  \
		.' /         \ `.
	    .-~.-~/           \~-.~-.
    .-~ ~    |             |    ~ ~-.
    `- .     |   CLIArmy   |     . -'     
         ~ - |             | - ~
             \             /
           ___\           /___
           ~;_  >- . . -<  _i~
              `'         `'

Good enough? :DDDDDD

**Comment by slaffcheff** [comment]
Visualize it (somehow) and we can use it, sure.

**Comment by teddy931130** [comment]
Prototypes...


![image uploaded from ios 1](https://user-images.githubusercontent.com/16972674/29520742-10103b18-868b-11e7-9b2f-53dfcbcd69ca.jpg)
![image uploaded from ios](https://user-images.githubusercontent.com/16972674/29520743-1037e906-868b-11e7-9f2a-894a292ca6fd.jpg)

**Comment by teddy931130** [comment]
![image uploaded from ios](https://user-images.githubusercontent.com/16972674/29526122-74883d18-869d-11e7-9b45-90127a10db82.jpg)

**Comment by slaffcheff** [comment]
add the newest proposal please

**Comment by teddy931130** [comment]
![image uploaded from ios](https://user-images.githubusercontent.com/16972674/29932891-5188e0a4-8e7e-11e7-8522-16eed63c99af.jpg)

**Comment by slaffcheff** [comment]
i was thinking .. this mp5 is too wide away. Can we switch the weapon with knife pointing up, vertically.

**Comment by teddy931130** [comment]
yeah, alright. I will tell the guy to modify it. Any knife preference? a small dagger, a machette, a sword?

**Comment by slaffcheff** [comment]
Rambo knife 

https://www.google.bg/search?q=rambo+knife&rlz=1C1CHZL_bgBG682BG682&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjm3sPgnYbWAhXCWhoKHTfoBhMQ_AUICigB&biw=1536&bih=735

**Comment by teddy931130** [comment]
![image uploaded from ios](https://user-images.githubusercontent.com/16972674/30263163-8aea885a-96dc-11e7-9353-4081f783b53c.jpg)
